shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -717,10 +717,24 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
                     event.setScrollX((int) position);
                     event.setMaxScrollX((int) max);
                 }
+                event.setItemCount(object.scrollChildren);
+                event.setFromIndex(object.scrollIndex);
+                int visibleChildren = object.childrenInHitTestOrder.size() - 1;
+                for (; visibleChildren >= 0; visibleChildren--) {
+                    SemanticsObject child = object.childrenInHitTestOrder.get(visibleChildren);
+                    if (!child.hasFlag(Flag.IS_HIDDEN)) {
+                        break;
+                    }
+                }
+                event.setToIndex(object.scrollIndex + visibleChildren);
                 sendAccessibilityEvent(event);
             }
-            if (object.hasFlag(Flag.IS_LIVE_REGION) && !object.hadFlag(Flag.IS_LIVE_REGION)) {
-                sendAccessibilityEvent(object.id, AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED);
+            if (object.hasFlag(Flag.IS_LIVE_REGION)) {
```

Comment:
This change is completely unrelated to the scroll change, right? I would put this into a separate PR.